run_with_login

```diff
@@ -0,0 +1,7 @@
+#! /bin/sh
+
+: "${APTIBLE_USERNAME?Need to set environment APTIBLE_USERNAME}"
+: "${APTIBLE_PASSWORD?Need to set environment APTIBLE_PASSWORD}"
```

Comment:
will these lines force the script to fail and exit with a non-zero return value? if not, we should probably add that in so the script exits early and fails the BuildKite / Jenkins build